src/fs.rs

```diff
@@ -507,6 +507,15 @@ impl Fs {
         &self.home_directory
     }
 
+    pub fn change_current_directory(&mut self, path: &str) {
+        if path.starts_with('/') {
+            self.current_directory = GuestPathBuf::from(GuestPath::new(&path));
+        } else {
+            self.current_directory = self.current_directory.join(path);
+        }
+        assert!(self.exists(&self.current_directory));
```

Comment:
Maybe make this into the return value? `Result<(), ()>` is a bit of a weird type but would clearly signal that it can fail.

Reply:
Done